src/views/kitchen-sink/page.php

```diff
@@ -1,4 +1,17 @@
 <?php
+/**
+ * The Events Calendar Styles kitchen sink.
+ *
+ * Displays all the styles applied to all UI components.
+ *
+ * @link http://m.tri.be/1aiy
+ *
+ * @version 4.2
+ *
+ * @package TribeEventsCalendar
+ *
+ */
```

Comment:
:no_entry_sign: Additional blank lines found at end of doc comment

<sub>hash: 5489785de8ca7780da6c5df35320eae5</sub>